src/components/side-panel/layer-panel/layer-panel.tsx

```diff
@@ -132,7 +146,7 @@ function LayerPanelFactory(LayerConfigurator, LayerPanelHeader) {
             isVisible={config.isVisible}
             label={config.label}
             labelRCGColorValues={config.dataId ? datasets[config.dataId].color : null}
-            layerType={layer.type}
+            layerType={layer.type!}
```

Comment:
what's the `!` here? is it a typo?

Reply:
[non-null-assertion-operator](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-)

layer should have type defined, isn't it?

Reply:
it's not always the case, a layer when first created, type might be `null`, `LayerPanelHeader` should handle `layerType: null` case

Reply:
ok, fixed